Balances/NicehashExternal.ps1

```diff
@@ -0,0 +1,77 @@
+﻿using module ..\Include.psm1
```

Comment:
Double check that the encoding is correct.

Reply:
ï»¿using module 

Encoding IS UTF-8 with BOM